ui/server/prisma/schema.prisma

```diff
@@ -155,6 +155,8 @@ model Round {
   statusAccount   Account? @relation(fields: [statusAccountId], references: [id])
   statusAccountId String?  @unique
 
+  inviteNonce   Int?
```

Comment:
Is Nonce a good name here? My understanding is that you're only supposed to use a nonce ("number once") once and then throw it away. Maybe not that important of a difference though.

Reply:
this number is only generated once. Once generated, this number can not be generated again for the same round (assuming max one link is generated in 1ms and clock not moved behind).

Reply:
I mean that to be called a cryptographic nonce https://en.wikipedia.org/wiki/Cryptographic_nonce I think we'd have to delete the nonce once we've sent it to a user or something, but here we might sent it to many many users. `inviteId` might make more sense in this case I think. No big deal but might make future developers less confused.

Reply:
User is not supposed to work with the nonce. They will share the invite links. I intentionally named it nonce, so if this *number* generation logic is changed, developer knows that this number should not be repeated. I think it is ok to keep like this.